non-critical-infra/hosts/umbriel.nixos.org/mailserver/default.nix

```diff
@@ -0,0 +1,34 @@
+{ config, pkgs, ... }:
+
+{
+  mailserver = {
+    enable = true;
+    certificateScheme = "acme-nginx"; #<<< TODO: I don't think this'll work: `pantalaimon` is listening on port 80 >>>
+
+    # Until we have login accounts, there's no reason to run either of these.
+    enablePop3 = false;
+    enableImap = false;
+
+    # Keep in sync with `iac/pulumi/app/dns.py`
+    fqdn = config.networking.fqdn;
+    # TODO: create an `MX` record for `playground.nixos.org` (value `umbriel.nixos.org`): <https://nixos-mailserver.readthedocs.io/en/latest/setup-guide.html#set-a-mx-record>.
+    # TODO: create an `SPF` record for `playground.nixos.org`: <https://nixos-mailserver.readthedocs.io/en/latest/setup-guide.html#set-a-mx-record>.
+    # TODO: create `DKIM` TXT record: <https://nixos-mailserver.readthedocs.io/en/latest/setup-guide.html#set-dkim-signature>.
+    #       (can't do this until after SNM is deployed)
+    # TODO: set a `DMARC` record: <https://nixos-mailserver.readthedocs.io/en/latest/setup-guide.html#set-a-dmarc-record>.
+    #       (and then make it strict (`v=DMARC1; p=reject; adkim=s; aspf=s;`) once things are working)
+    # TODO: change to `nixos.org` when ready
+    domains = [ "playground.nixos.org" ];
```

Comment:
@Mic92, I was thinking we could first set this up end to end with some other domain than `nixos.org` so we can test it out without breaking existing `nixos.org` mail. Perhaps you have a different idea for testing this out?